CI/travis.validate_deployment.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ -z "${TRAVIS_TAG}" ]; then
+  exit 0
+fi
+
+function validate_qmake() {
+  VALID_QMAKE=$(cat src/mudlet.pro | pcregrep --only-matching=1 "^VERSION ? = ?(\d+\.\d+\.\d+)$")
+
+  if [ -z "${VALID_QMAKE}" ]; then
+  echo "mudlet.pro isn't set to a valid version in a release build."
+  exit 1
+  fi
+}
+
+function validate_cmake() {
+  VALID_CMAKE=$(cat CMakeLists.txt | pcregrep --only-matching=1 "set\(APP_VERSION (\d+\.\d+\.\d+)\)$")
+
+  if [ -z "${VALID_CMAKE}" ]; then
+    echo "CMakeLists.txt isn't set to a valid version in a release build."
+    exit 1
+  fi
+}
+
+function validate_updater_environment_variable() {
+  if [ "$WITH_UPDATER" != "NO" ]; then
+     echo "Updater is disabled in a release build."
+     exit 1
```

Comment:
I thought WITH_UPDATER=NO turned the updater off, shouldn't we be checking if it == "NO" ?

Reply:
Thanks for the catch!